### PR TITLE
Internationalization with next-i18next

### LINKED ISSUE #28: introduce localization
Todo
- [x] decide translation strategy 
- [ ] introduce
- [ ] translate as community tasks

### PR DESCRIPTION
**Pull request by alecrem**
Closes #28

- Setting up next-i18next
- Front page localized

### CODE REVIEW

**Review by geeknees — APPROVED** [review]
It looks so good. I thought it would be better if the switch button was like a dark mode instead of a link. nice!

**Comment by alecrem** [comment]
The switch... Yes, I thought the same but it's hard to find an icon for switching languages that people will actually understand, and that is not a country's flag. Not even on fontawesome!

I think this is mergeable even if the translation is not complete, but I would like to make a Link and a Button component that have a next/link wrapper and a chakra-ui component inside, so that our pages are cleaner and easier to maintain.

**Comment by geeknees** [comment]
`🇺🇸 / 🇯🇵` 

It's not a good idea to link a country to a language, but how about emoji for a start?

> However, associating a language with a flag may cause problems...
    Arabic is also used outside the United Arab Emirates https://www.quora.com/Which-flag-represents-the-Arabic-language
    English is also used in the US and UK
    Perhaps Telugu and Marathi are both used in India


ref. https://github.com/snapshot-labs/snapshot/pull/1232

**Comment by alecrem** [comment]
Personally, I'd rather keep the language name than use flags.

Languages + flags = 💣

File: public/locales/ja/common.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "MINT_YOUR_KAMON_HEADING": "家紋をミント",
+  "MINT_YOUR_KAMON_EXPLANATION": "家紋NFTはHenkakuコミュニティへの参加を証明する",
```
Comment (hourglasshoro) [inline public/locales/ja/common.json:3]: [IMO] 
You might want to adjust words tone. like する->します
家紋NFTはHenkakuコミュニティに参加していることを証明します

**Review by hourglasshoro — APPROVED** [review]
LGTM!

File: public/locales/ja/common.json (hunk 1 above)
Comment (alecrem) [reply public/locales/ja/common.json]: That's right! I just had a look at all the strings and that's the only place I wrote it like that. Of course, it was the first one lol

Hopefully nothing should be critical, but any corrections are always welcome!

**Comment by alecrem** [comment]
The only thing that is not on i18n yet are the error messages regarding the `ipfs` API request.

I'm happy to track that as #70 and merge this any time now, if anyone has a minute to check the deployed version on vercel.

**Review by SamuraiT — APPROVED** [review]
Cool ! good Work!!

**Pull request merged**